heartcheck-redis.gemspec

```diff
@@ -17,10 +17,12 @@ Gem::Specification.new do |spec|
   spec.test_files = spec.files.grep(/^spec\//)
   spec.require_paths = ['lib']
 
+  spec.required_ruby_version = '>= 2.2.2'
+
   spec.add_runtime_dependency 'net-telnet', '~> 0.1.1'
 
   spec.add_dependency 'heartcheck', '>= 1.0.0'
-  spec.add_dependency 'redis', '>= 3.2.0', '< 3.4.0'
+  spec.add_dependency 'redis', '>= 3.2.0', '< 5'
```

Comment:
It would be nice if it was possible to run travis with different versions of redis (3, 4 and 5) to check if this code is compatible with all of them.

I don't know if it is possible, do you know?

@rhruiz do you know?